src/main/java/com/faforever/client/vault/VaultEntityController.java

```diff
@@ -69,6 +69,7 @@
   public Pagination pagination;
   public Button lastPageButton;
   public Button manageModsButton;
+  public Button mapsManagementButton;
```

Comment:
Rather than adding a new management button to the abstract class make one of them abstract and set the actions and text in the subclass. Similar to the upload button. I like the placement you have for the maps one.

Reply:
Done. Yes, I like this placement.